src/main/kotlin/no/nav/syfo/domain/Sykmelding.kt

```diff
@@ -0,0 +1,118 @@
+package no.nav.syfo.domain
+
+import no.nav.syfo.aksessering.api.ArbeidsgiverDTO
+import no.nav.syfo.aksessering.api.BehandlingsutfallDTO
+import no.nav.syfo.aksessering.api.BehandlingsutfallStatusDTO
+import no.nav.syfo.aksessering.api.GradertDTO
+import no.nav.syfo.aksessering.api.PeriodetypeDTO
+import no.nav.syfo.aksessering.api.RegelinfoDTO
+import no.nav.syfo.aksessering.api.SykmeldingDTO
+import no.nav.syfo.aksessering.api.SykmeldingsperiodeDTO
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class Sykmelding(
+    val id: String,
+    val bekreftetDato: LocalDateTime?,
+    val behandlingsutfall: Behandlingsutfall,
+    val legekontorOrgnummer: String,
+    val legeNavn: String?,
+    val arbeidsgiver: Arbeidsgiver?,
+    val sykmeldingsperioder: List<Sykmeldingsperiode>
+)
+
+data class Behandlingsutfall(
+    val ruleHits: List<Regelinfo>,
+    val status: BehandlingsutfallStatus
+)
+
+data class Regelinfo(
+    val messageForSender: String,
+    val messageForUser: String,
+    val ruleName: String
+)
+
+enum class BehandlingsutfallStatus {
+    OK, MANUAL_PROCESSING, INVALID
+}
+
+data class Arbeidsgiver(
+    val navn: String,
+    val stillingsprosent: Int
+)
+
+data class Sykmeldingsperiode(
+    val fom: LocalDate,
+    val tom: LocalDate,
+    val gradert: Gradert?,
+    val behandlingsdager: Int?,
+    val innspillTilArbeidsgiver: String?,
+    val type: Periodetype
+)
+
+data class Gradert(
+    val grad: Int,
+    val reisetilskudd: Boolean
+)
+
+enum class Periodetype {
+    AKTIVITET_IKKE_MULIG,
+    AVVENTENDE,
+    BEHANDLINGSDAGER,
+    GRADERT,
+    REISETILSKUDD,
+}
+
+// Konverter Sykmelding til SykmeldingDTO
```

Comment:
Trenger du kommentaren her?

Reply:
neida... jeg bare følte for å ha noe som sepparerte dataklassene fra konverteringsfunksjonene.